.github/workflows/test.yml

```diff
@@ -118,6 +118,23 @@ jobs:
           fi
         env:
           RUFF_VERSION: ${{ steps.ruff-action.outputs.ruff-version }}
+  test-default-version-from-pyproject-poetry-groups:
```

Comment:
One more thing I forgot. Can you please add this test to the needs section of the all-tests-passed jobs around line 280?

Reply:
Good catch! Really should've noticed that one myself 😅